index.html

```diff
@@ -314,18 +318,18 @@ <h4>Multikey</h4>
           </pre>
 
           <p>
-            The `secretKeyMultibase` property represents a Multibase-encoded Multikey
-            expression of an Ed25519 secret key (sometimes also referred to as a private key).
-            The value starts with the two-byte prefix `0x8026` (the varint expression of `0x1300`),
-            followed by the 32-byte Ed25519 secret key data. The combined 34-byte
-            value is then base58-btc encoded and `z` is added as the prefix on the
-            encoded value.
+The `secretKeyMultibase` property represents a Multibase-encoded Multikey
+expression of an Ed25519 secret key (sometimes also referred to as a private
+key). The value starts with the two-byte prefix `0x8026` (the varint expression
+of `0x1300`), followed by the 32-byte Ed25519 secret key data. The combined
```

Comment:
There doesn't *appear* to be any reason why readers would care that _`0x8026` [is] the varint expression of `0x1300`_, so I'm thinking that should be left out, or explained at least a bit more.
```suggestion
key). The value starts with the two-byte prefix `0x8026`,
followed by the 32-byte Ed25519 secret key data. The combined
```